notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -917,6 +917,55 @@ what if I was trapped inside a game [Ash discovered that the only way to exit th
 the legend of King Arthur [A priest recanted the story of how a dying King Arthur was taken by ship to the isle of Avalon to see the fairy Morgan le Fay.]
 
 
+movie: Evolution (2001)
+=======================
+
+:: Title
+Evolution
+
+:: Description
+Evolution is a 2001 American comic science fiction film directed by Ivan
+Reitman.
+
+Synopsis: A pair of college professors investigate a meteorite crash in
+Arizona. They discover that the meteorite harbors extraterrestrial lifeforms,
+which are evolving very quickly into large, diverse and outlandish creatures.
+
+:: Date
+2001-06-08
+
+:: References
+https://en.wikipedia.org/wiki/Evolution_(2001_film)
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+invasive life form from outer space [The main novelty of the film is that some alien microbes hitched a ride to the Arizona desert on a meteor, began to evolve at an accelerated rate, and soon thereafter gave rise to complex creatures that seemed destined to overrun the entire Earth.]
+
+:: Major Themes
+the scientist profession [The story follows the local college science professors Ira Kane and Harry Block as they study newly discovered, fast-evolving extraterrestrial microbes that had hitched a ride to Earth on a meteor.]
+super accelerated Darwinian evolution [The extraterrestrial microbes evolved into complex dinosaur-like creatures in just a handful of days.]
+romantic love [The story built up to Ira and Allison falling in love in the end.]
+
+:: Minor Themes
+what it is like in university [The viewer is shown various stereotypical community college happenings: An apathetic Professor Ira Kane gave A's to nearly everyone in his biology class on their biology research projects, there was a wild college party, etc.]
+genetics [Ira discovered that the extraterrestrial life was nitrogen-based and had DNA that consisted of ten base pairs.]
+geology [Professor Harry Block made a point that he was a representative of the U.S. Geological Survey when he went to sample a recently crashed meteor.]
+brother and brother [The brothers Deke and Danny Donald were two of the dumbest students in Dr. Ira Kane's biology class.]
```

Comment:
Let me just point out that the comment doesn't actually state that we saw anything about the two brothers' relationship with each other.

Reply:
Indeed. I suppose we can do a bit better than that. New and improved motivation added.